chaussette/server.py

```diff
@@ -166,6 +166,10 @@ def main():
                     args.backend)
         sys.exit(0)
 
+    # make the arg parser available from the loaded wsgi module via
+    # the sys module, so it can more easily retrieve the 'arguments'
+    # given on chaussette cmd line
+    sys.argp = args
```

Comment:
patching sys seems not robust. Why not an environement variable ? like CHAUSSETTE_PARSED_ARGS or something like this.